### PR TITLE
Fix exception class

### PR DESCRIPTION
**Pull request by trompette**
Dear maintainer,

Here is a PR with tests to fix a typo I found in the code base.

Cheers!

### CODE REVIEW

File: Tests/MockTest.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+namespace Danhunsaker\Geocoder\Provider\Mock\Tests;
+
+use Danhunsaker\Geocoder\Provider\Mock\Mock;
+use Http\Client\HttpClient;
+use PHPUnit\Framework\TestCase;
+
+class MockTest extends TestCase
+{
+    public function testInstanciation()
```
Comment (danhunsaker) [inline Tests/MockTest.php]: *Instantiation

File: Tests/MockTest.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                ],
+                'country' => 'United States',
+                'countryCode' => 'US',
+            ]
+        );
+
+        $this->assertInstanceOf(Mock::class, $mock);
+    }
+
+    public function testInstanciationWithEmptyCoordinates()
```
Comment (danhunsaker) [inline Tests/MockTest.php]: Same as above.

File: Tests/MockTest.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                ],
+                'country' => 'United States',
+                'countryCode' => 'US',
+            ]
+        );
+
+        $this->assertInstanceOf(Mock::class, $mock);
+    }
+
+    public function testInstanciationWithEmptyAddress()
```
Comment (danhunsaker) [inline Tests/MockTest.php]: Same again.

File: Mock.php [hunk 4]
```diff
@@ -52,11 +52,11 @@ public function __construct(HttpClient $client, $latLong, $address)
         $this->address = $address;
 
         if (empty($this->latLong)) {
-            throw new \InvalidArgument('The Lat/Long must be set to use the Mock provider.');
+            throw new \InvalidArgumentException('The Lat/Long must be set to use the Mock provider.');
```
Comment (danhunsaker) [inline Mock.php]: I think the real typo here is actually in not using the `InvalidArgument` from Geocoder itself: `Geocoder\Exception\InvalidArgument`. My oversight.

**Review by danhunsaker — CHANGES_REQUESTED** [review]
Some minor quibbles, but otherwise looks good!

File: Tests/MockTest.php (hunk 1 above)
Comment (trompette) [reply Tests/MockTest.php]: Sorry, I used the French word... :man_shrugging:

**Comment by trompette** [comment]
@danhunsaker, comments fixed!

**Review by danhunsaker — APPROVED** [review]

**Pull request merged**